search-services/latest/using/index.md

```diff
@@ -358,12 +358,11 @@ These examples show how to embed queries in CMIS.
 
 ```sql
 - strict queries
-SELECT * FROM Document WHERE CONTAINS("zebra")
 SELECT * FROM Document WHERE CONTAINS("quick")
 
 - Alfresco extensions
-SELECT * FROM Document D WHERE CONTAINS(D, 'cmis:name:'Tutorial")
-SELECT cmis:name as BOO FROM Document D WHERE CONTAINS('BOO:'Tutorial")
+SELECT * FROM Document D WHERE CONTAINS(D, 'cmis:name:\'Tutorial\'')
+SELECT cmis:name as BOO FROM Document D WHERE CONTAINS('BOO:\'Tutorial\'')
```

Comment:
FYI: In the ACS 5.2 docs, this is now shown in line https://github.com/Alfresco/docs-alfresco/blob/master/content-services/5.2/develop/alfresco-full-text-search-ref.md?plain=1#L379 (I made a change the day after your PR which removed a chuck of text).